tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = py36, py37, py36-coverage
 [testenv]
 deps =
     -r{toxinidir}/requirements.txt
+    -r{toxinidir}/unittests/test_requirements.txt
```

Comment:
For testing purpose, I think we should use only `test-requirements.txt`... can we make the necessary changes on these lines.

Also, let's remove `black, isort` and others that are not required for execution of the framework from `requirements.txt`

Reply:
done